engines/sludge/variable.cpp

```diff
@@ -313,6 +313,69 @@ bool Variable::loadStringToVar(int value) {
 	return (bool)(varData.theString != NULL);
 }
 
+// use this in debug contexts to avoid loading objects
+Common::String Variable::peekTextFromAnyVar() const {
```

Comment:
Please remove this method, I do not understand why is it here, so massively duplicating the code.

Just pass to `getTextFromAnyVar()` the `skipLoad` parameter.